### PR TITLE
Fixed the textual error and some css errors on cards and images

### PR DESCRIPTION
**Pull request by nshanbhag20001412**
- Fix1
   image
  - The color variable naming was not done properly so fixed it. 
    image

- Fix2
  - The documentation of card had some extra classes applied which was not required.
 
- Fix 3
  - Changed image's object-fit property to cover.

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Good to merge!

File: Get Started/getstarted.html [hunk 1]
```diff
@@ -41,19 +41,19 @@ <h3>Colors</h3>
               <span >Primary Color<br>var(--primary)</span>
             </div>
             <div class=" theme-color-block off-white"  >
```
Comment (sanathkumar26700) [inline Get Started/getstarted.html:43]: Could use BEM naming for easy understanding of class names.

**Review by sanathkumar26700 — APPROVED** [review]
Everything else is fine!

File: Get Started/getstarted.html (hunk 1 above)
Comment (nshanbhag20001412) [reply Get Started/getstarted.html:43]: okay

**Pull request merged**